CHANGELOG.md

```diff
@@ -26,14 +30,26 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
     - The copyright for all sources is now attributed to Open Enclave SDK contributors.
 - Update Intel DCAP library dependencies to 1.4.1.
 - Update Intel PSW dependencies to 2.6.100.2 on Windows.
-- The stack guard for OE SDK on SGX is turned on. It is also turned on for enclaves.
+- Enable /W2 to treat W2 warnings as errors for Windows builds. Additionally, fix errors which arise from enabling this flag.
+- Fix oe_random for size > 1024.
+- Add openenclave namespace to dl and crypto libraries. Fixes #2082
+- Implement host side sockets on Windows.
+- Removed code related to deprecation of strftime.
+- Stack guard for enclaves is enabled. Enclave libs and enclaves are built using `-fstack-protector-strong`.
+- Enclave libs and enclaves are built using `-gc-sections`.
+- Overhaul OCaml build system to use esy and upgraded OCaml version with reproducibility.
```

Comment:
I would rephrase this in terms of impact to the user, which is that the OE SDK project now has a new set of dependencies that a contributor would need to refresh their setup with. Some of @andschwa instructions might be useful here.

Reply:
I don't think we've been focusing on "impact to user" in this changelog. For instance, several of the above changelog items are of a similar nature to this one.

We're bumping a new version; I would expect devs of the project to expect breaking changes. However, it's only going to apply if they've been building from source, but only from the last release branch, not from master. The instructions to build and run should be sufficient. (We could call out that they will want to review the build instructions if they're building from source and haven't pulled since the last release.)

Reply:
@CodeMonkeyLeet @andschwa 
This is the wording I am considering:
- Replace OCPWin and OCaml with esy. T
   - The CMake-driven OCaml build is replaced with esy and dune. To install esy(which is now a prerequisite to building the SDK from source):
      - On Linux, `sudo ansible-playbook oe-linux-esy-setup.yml`
      - On Windows, `npm install -g esy@0.5.8`

Reply:
This is the PR I referenced: https://github.com/openenclave/openenclave/pull/2221